aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/KafkaEvent.java

```diff
@@ -42,6 +42,7 @@
         private String timestampType;
         private String key;
         private String value;
+        private List<Map<String, byte[]>> headers;
```

Comment:
Would this be better as a `String` rather than a `byte[]`?

Reply:
The byte[] is [defined by Kafka](https://kafka.apache.org/26/javadoc/index.html?org/apache/kafka/common/header/Headers.html), I think we should pass what we got from Kafka to customer.